### PR TITLE
Add A General Templates Document

### PR DESCRIPTION
**Pull request by Githubbubber**
May be closed, depending on how and where the templates will be placed.

A summary of all of the templates we will be using in our Queer Global development work.

Send me a message in Slack if needed.

### CODE REVIEW

**Review by mnyon-grandkru — COMMENTED** [review]
Is the intention to outline what will be in separate documents and then create that distinct document here? Like will the Changelog Entry template be CHANGELOG_ENTRY.md?

**Comment by Githubbubber** [comment]
> 

Great question! I put them all in one file for ease. I’ll take a poll in Slack.

File: GENERAL_TEMPLATES.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# Templates
+Deliverable
```
Comment (Githubbubber) [inline GENERAL_TEMPLATES.md:2]: To be disregarded; Is from reference

File: GENERAL_TEMPLATES.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  Test Cases / Results
+  Test Coverage Reports
+  GForge Bug tracker for bug reporting
+Reports
+Test results report 
+  Test Final Report - Sign-Off
+
+---
+
+## Image File Naming Best Practices
```
Comment (Githubbubber) [inline GENERAL_TEMPLATES.md:18]: "Image File Naming Best Practices” -> Perhaps be done in a pre-commit hook? NPM module to take care of this.

Comment (Githubbubber) [reply GENERAL_TEMPLATES.md:11]: Made an [item to work on this](https://github.com/orgs/QueerGlobal/projects/3?pane=issue&itemId=32344223) in the [Infrastructure Roadmap](https://github.com/orgs/QueerGlobal/projects/3).

**Review by chris-watson — APPROVED** [review]
Thanks for taking care of this!

**Pull request merged**